src/app/prt-staking/layout.tsx

```diff
@@ -0,0 +1,26 @@
+import { Providers } from '@/app/providers'
+import { LeverageBanner } from '@/components/banners/leverage-banner'
+import { Footer } from '@/components/footer'
+import { Header } from '@/components/header'
+import { isLeverageSuiteEnabled } from '@/feature-flags'
+
+type Props = {
+  children: React.ReactNode
+}
+
+export const metadata = {
+  title: 'PRT Staking',
+}
+
+export default function Layout({ children }: Props) {
+  return (
+    <div className="flex h-fit flex-col bg-[url('/presale-splash.jpg')] bg-cover">
+      <Providers>
+        {isLeverageSuiteEnabled() && <LeverageBanner />}
```

Comment:
Should this be `isStakingEnabled`?

Reply:
Ah no but don't you remove this in the next PR?

Reply:
Yep - removed in the next PR, merged that in here